apps/mobile/src/components/react-hook-form/native-base/rhf-list-selector.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button } from 'native-base';
+import React from 'react';
+import { useFormContext, useWatch } from 'react-hook-form';
+
+type Option = {
+  label: string;
+  value: string;
+};
+
+type RHFNativeBaseListSelectorProps = {
+  name: string;
+  options: Option[];
+};
+
+function RHFNativeBaseListSelector({ name, options }: RHFNativeBaseListSelectorProps) {
```

Comment:
Added the rule with `warn` for now, otherwise, I'd have to change a thousand of files. I'll convert the `function` declarations into arrow format in the next PR.